src/components/DeatilCard/DetailCard.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link, useSearchParams } from 'react-router-dom';
```

Comment:
to test this component, you need to install and use **jest** to mock data, functions and external dependencies